alert-policies/ansible-automation-controller/JobsPending.yml

```diff
@@ -0,0 +1,36 @@
+name: Jobs Pending
+
+description: |+
+  This alert is triggered when the jobs pending is higher than 1 during 5 minutes.
```

Comment:
```suggestion
  This alert is triggered when the jobs pending is higher than 1 within 5 minutes.
```

Reply:
Hi @**paperclypse** ,
Reviewed text correction has been changed. Kindly review it.